src/webmon_app/reporting/dasmon/views.py

```diff
@@ -36,8 +36,12 @@ def dashboard(request):
     """
     # Get the system health status
     global_status_url = reverse(settings.LANDING_VIEW, args=[])
+    instrument_data = view_util.get_instrument_status_summary()
+    cutoff = int(len(instrument_data) / 2)
     template_values = {
-        "instruments": view_util.get_instrument_status_summary(),
+        "instruments": instrument_data,
```

Comment:
It is better to add some documentation here.

Reply:
do you mean a comment in the code explaining the intent of the different dictionary keys?

Reply:
> Besides the in-line comment, one more general question: is there any way to create a test to reflect the requirement?

The only check that comes to mind if to verify that function `dashboard()` can populate template `dahsboard.html` if appropriate template_values are passed on.
I see there are already some tests under `tests/` that check the Views of some pages so I could take a look to these and see if I can come up with a similar View test for the "extended dashboard" page. Does that sound good?

Reply:
A comment such that the people in the future can easily understand that the instruments are put to 2 columns.